TESTING.md

```diff
@@ -1,6 +1,153 @@
-# VSCode testing
+# ACPaaS UI Testing Guide
 
-To make life easier in VScode for testing with Jest.
-We suggest you install the following plugin:
+When developing, testing is a major part of the work.
+We use [Jest](https://jestjs.io) in combination with [Enzyme](https://airbnb.io/enzyme) as our test suite.
 
-https://github.com/jest-community/vscode-jest
\ No newline at end of file
+## VSCode testing
```

Comment:
I'm not sure if this section is as important as the next one. I would suggest to move this info to the bottom or to use no heading.

Reply:
Removed the heading.